src/plan/generational/global.rs

```diff
@@ -178,42 +178,42 @@ impl<VM: VMBinding> Gen<VM> {
     }
 
     /// Trace objects for spaces in generational and common plans for a full heap GC.
-    pub fn trace_object_full_heap<T: TransitiveClosure>(
+    pub fn trace_object_full_heap<Q: ObjectQueue>(
```

Comment:
Shall we mark this as `#[inline(always)]`? Also the `trace_object_nursery` function below as well

Reply:
I tried.  However, if I mark this as `#[inline(always)]`, the Rust compiler will decide not to inline another call inside this function.  Then I marked another function as `#[inline(always)]`, and then the Rust compiler will again decide not to inline yet another function.  And this went on and on.  So I decided to fix the inlining problem later, not in this PR, because the performance of generational GC doesn't seem to be affected by this PR.